internal/util/umount.go

```diff
@@ -0,0 +1,63 @@
+// Copyright 2021 Red Hat, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package util
+
+import (
+	"fmt"
+	"os/exec"
+	"time"
+
+	"golang.org/x/sys/unix"
+)
+
+// runGetExit runs the command and returns the exit status. It only returns an error when execing
+// the command encounters an error. exec'd programs that exit with non-zero status will not return
+// errors.
+func runGetExit(cmd string, args ...string) (int, string, error) {
+	tmp, err := exec.Command(cmd, args...).CombinedOutput()
+	logs := string(tmp)
+	if err == nil {
+		return 0, logs, nil
+	}
+	exitErr, ok := err.(*exec.ExitError)
+	if !ok {
+		return -1, logs, err
+	}
+	status := exitErr.ExitCode()
+	return status, logs, nil
+}
+
+func UmountPath(path string) error {
```

Comment:
Now that we have this, the blackbox tests should use it instead of having its own copy.

Not a blocker.